posts/201804.md

```diff
@@ -65,14 +78,15 @@ Finally,
 
 > viniciusfrias: We're excited to announce PagueCripto.com, a Brazilian crypto-to-fiat payment gateway which accepts Decred among other cryptocurrencies for Brazilians to pay daily bills, such as credit cards, energy, rent, etc, and also to make local bank transfers. Our service is both a web platform and an Android app, and as our community is relevant in Brazil, we are offering a discount coupon (50%) in service fees using DCR until May 14, 2018. Check it out at [paguecripto.com](https://www.paguecripto.com/) and in Google Play Store. ([slack](https://decred.slack.com/archives/C0NRMDVMJ/p1525043419000038))
 
-[changenow.io](https://changenow.io/), a non-custodian exchange for fast conversions [added](https://twitter.com/decredproject/status/981623403378454528) DCR.
-
 > michae2xl: Decred is now available on [@ezBtcCanada](https://twitter.com/ezBtcCanada) — an exchange with DCR/CAD trading pair. From Toronto — https://www.ezbtc.ca/trade/alt-coins
 
 i2Rav announced a new [trading desk](https://i2trading.com/) that will be offering DCR/EUR pair.
 
 > Eventually we hope to offer the pair in GBP and YEN as well
 
+[changenow.io](https://changenow.io/), a non-custodian exchange for fast conversions, [added](https://twitter.com/decredproject/status/981623403378454528) DCR.
+#reasoning for move: the only one in the list not about a fiat pair
+
```

Comment:
Interesting point of view. I tried to sort paragraphs by date things occured. This way as you read every section you travel in time from beginning to the end of the month. I made an exception in Media section, where I placed things made by Decred community on top (3 of them) and everything else later. This instant exchange being the only without fiat pair somewhat breaks the "fiat group". So the order in your version would be (fiat bridges ordered by date), (non-fiat bridges ordered by date).

I don't have an opinion at this point so I'd accept it.